components/availability/src/index.html

```diff
@@ -203,6 +216,26 @@
           False
         </label>
       </fieldset>
+
+      <fieldset>
+        <legend>Max Bookable Slots</legend>
+        <label>
+          <input type="number" name="max-slots" value="1" />
```

Comment:
Can a `min` attribute be set for this? I can select negative numbers

Reply:
Added, but noting that the component works with negative numbers just fine ("You're not allowed to make more than -5 events" is still technically enforced :D )